jmclient/jmclient/wallet_rpc.py

```diff
@@ -551,6 +554,23 @@ def check_daemon_ready(self):
             raise BackendNotReady()
         return (daemon_serving_host, daemon_serving_port)
 
+    def get_wallet_cls_from_type(self, wallettype: str) -> BaseWallet:
+        if wallettype == "sw":
+            return SegwitWallet
+        elif wallettype == "sw-legacy":
+            return SegwitLegacyWallet
+        elif wallettype == "sw-fb":
+            return SegwitWalletFidelityBonds
+        else:
+            raise InvalidRequestFormat()
+
+    def get_wallet_name_from_req(self, walletname):
```

Comment:
```suggestion
    def get_wallet_name_from_req(self, walletname: str) -> str:
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1461/commits/7f4eaa9846f2f489480de1f2f8bfa9a1e2e96686